recipe/meta.yaml

```diff
@@ -35,9 +37,13 @@ test:
 
 about:
   home: https://pypi.org/project/findpython/
+  description: A utility to find python versions on your system
```

Comment:
Maybe just copy the upstream description (without the "2@' symbol)

Reply:
This has been updated